src/lib/tradingView/dydxfeed/index.ts

```diff
@@ -248,7 +248,7 @@ export const getDydxDatafeed = (
       onRealtimeCallback,
       subscribeUID,
       onResetCacheNeededCallback,
-      lastBar: lastBarsCache.get(`${symbolInfo.full_name}/${RESOLUTION_MAP[resolution]}`),
+      lastBar: lastBarsCache.get(`${symbolInfo.symbol}/${RESOLUTION_MAP[resolution]}`),
```

Comment:
i dont think `.symbol` exists? also lol at having `any` types up there lol

Reply:
Ah will change to `ticker` For some reason I thought `symbolInfo` was the return from `getSymbol` function. Thanks for catching

Reply:
i guess `full_name` doesnt exist either 😆